nilearn/plotting/surf_plotting.py

```diff
@@ -1626,7 +1626,7 @@ def plot_surf_roi(surf_mesh, roi_map, bg_map=None,
     display = plot_surf(mesh, surf_map=roi, bg_map=bg_map,
                         hemi=hemi, view=view, engine=engine,
                         avg_method='median', threshold=threshold,
-                        cmap=cmap, symmetric_cmap=False,
+                        cmap=cmap,
```

Comment:
probably want to make this smarter and set this to False unless symmetric_map is passed as a kwarg